src/components/molecules/AccountTeaser.module.css

```diff
@@ -0,0 +1,41 @@
+.blockies {
+  aspect-ratio: 1/1;
+  width: 13%;
+  height: 13%;
+  border-radius: 50%;
+  margin-left: 0;
+  margin-right: calc(var(--spacer) / 4);
+}
+
+.teaser {
+  max-width: 40rem;
+  height: 100%;
+}
+
+.link {
+  composes: box from '../atoms/Box.module.css';
+  padding: calc(var(--spacer) / 2) !important;
+  font-size: var(--font-size-mini);
+  height: 90%;
+  color: var(--color-secondary);
+  position: relative;
+}
+
+.link span {
+  font-size: var(--font-size-large);
+  margin-right: calc(var(--spacer) / 3);
+  margin-top: calc(var(--spacer) / 4);
+}
+.name {
+  margin-bottom: 0;
+  font-size: var(--font-size-base) !important;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+}
+
+.sales {
+  font-size: small;
+}
```

Comment:
I think it might look better to remove the margin we get o sales from `p` element, and use flex to center the name and number of sales, something like this
![Screenshot 2021-11-16 at 10 44 44](https://user-images.githubusercontent.com/37236152/141952481-2e762a2a-1d76-4eb4-b3c0-0ddf930525e5.png)
: